### PR TITLE
feat: mobile replace menu with fab

### PR DESCRIPTION
**Pull request by zhouhanseng**
part of https://github.com/DimensionDev/Maskbook/issues/1896

The UI details will be decided later.

### CODE REVIEW

File: packages/maskbook/src/extension/options-page/DashboardRouters/Container.tsx [hunk 1]
```diff
@@ -150,77 +142,65 @@ const useStyles = makeStyles((theme) => {
...
+            display: 'flex',
+            justifyItems: 'center',
+            alignItems: 'center',
+            marginBottom: theme.spacing(2),
+        },
     })
 })
 
 export default function DashboardRouterContainer(props: DashboardRouterContainerProps) {
-    const { title, actions, children, padded, empty, compact = false, leftIcons = [], rightIcons = [] } = props
```
Comment (zhouhanseng) [inline packages/maskbook/src/extension/options-page/DashboardRouters/Container.tsx:167]: leftIcons is no longer used.

File: packages/maskbook/src/extension/options-page/DashboardRouters/Container.tsx [hunk 2]
```diff
@@ -150,77 +142,65 @@ const useStyles = makeStyles((theme) => {
...
-                                          key: index,
-                                          size: 'small',
-                                          className: classes.titleIcon,
-                                      }),
-                                  )
-                                : null}
-                            {xsMatched ? null : (
-                                <div className={classes.buttons}>
-                                    {actions?.map((action, index) => React.cloneElement(action, { key: index }))}
+                        {Flags.has_native_nav_bar ? null : (
```
Comment (zhouhanseng) [inline packages/maskbook/src/extension/options-page/DashboardRouters/Container.tsx:180]: Using `Flags` on platform related place.

File: packages/maskbook/src/extension/options-page/DashboardRouters/Container.tsx [hunk 3]
```diff
@@ -150,77 +142,65 @@ const useStyles = makeStyles((theme) => {
...
+                                            {actions?.map((action, index) =>
+                                                React.cloneElement(action, { key: index }),
+                                            )}
+                                        </div>
+                                    )}
+                                </section>
+                                <div
+                                    className={classNames({
+                                        [classes.dividerPadded]: padded !== false,
+                                        [classes.dividerCompact]: xsMatched,
```
Comment (zhouhanseng) [inline packages/maskbook/src/extension/options-page/DashboardRouters/Container.tsx:198]: `useMatchXS` on flexible UI.

File: packages/maskbook/src/extension/options-page/DashboardRouters/Container.tsx [hunk 4]
```diff
@@ -150,77 +142,65 @@ const useStyles = makeStyles((theme) => {
             },
         },
         dividerCompact: {
-            [theme.breakpoints.down('sm')]: {
```
Comment (zhouhanseng) [inline packages/maskbook/src/extension/options-page/DashboardRouters/Container.tsx:153]: already be checked: https://github.com/DimensionDev/Maskbook/pull/1912/files#diff-af89c552a9b1289c018f0ee44fa5c8f1726cb0acd8e5e4330fcd65db8a9d69f3R198

**Review by Jack-Works — APPROVED** [review]

**Review by Jack-Works — APPROVED** [review]

**Pull request merged**